gapic-generator/lib/google/gapic/schema/wrappers.rb

```diff
@@ -261,26 +259,26 @@ def initialize descriptor, address, docs, input, output
           @output = output
         end
 
-        # @return [Array<Google::Api::MethodSignature>] The parameter lists
+        # @return [Array<String>] The parameter lists
```

Comment:
Similar to `scopes`, should this be at the same level of abstraction as the proto or should we break this down so the caller doesn't need to know it's `,` delimitated? (also I think the comment is outdated and `signature.proto` was moved or renamed)

Reply:
Updated.